integration_test/chart_values/backend.yaml

```diff
@@ -0,0 +1,2002 @@
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    # -- Ory KETO version
+    tag: "v0.6.0-alpha.1-sqlite"
+
+  imagePullSecrets: []
+
+  serviceAccount:
+    # -- Specifies whether a service account should be created
+    create: true
+    # -- Annotations to add to the service account
+    annotations: {}
+    # -- The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  podAnnotations: {}
+
+  podSecurityContext: {}
+    # fsGroup: 2000
+
+  # https://github.com/kubernetes/kubernetes/issues/57601
+  automountServiceAccountToken: true
+
+  # -- Default security context configuration
+  securityContext:
+    capabilities:
+      drop:
+      - ALL
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 100
+    allowPrivilegeEscalation: false
+    privileged: false
+
+  job:
+    annotations: {}
+
+  ingress:
+    read:
+      enabled: true
+      className: ""
+      annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+      hosts:
+        - host: keto.local
+          paths:
+            - path: /read
+              pathType: Prefix
+      tls: []
+      #  - secretName: keto-tls
+      #    hosts:
+      #      - keto.local
+    write:
+      enabled: false
+      className: ""
+      annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+      hosts:
+        - host: keto.local
+          paths:
+            - path: /write
+              pathType: Prefix
+      tls: []
+      #  - secretName: keto-tls
+      #    hosts:
+      #      - keto.local
+
+  service:
+    read:
+      enabled: true
+      type: ClusterIP
+      name: http-read
+      port: 80
+    write:
+      enabled: true
+      type: ClusterIP
+      name: http-write
+      port: 80
+
+  secret:
+    # -- Switch to false to prevent creating the secret
+    enabled: true
+    # ... and choose a different name for a secret you provide like this
+    # nameOverride: "MyOtherName"
+    secretAnnotations:
+      # Create the secret before installation, and only then. This saves the secret from regenerating during an upgrade
+      # pre-upgrade is needed to upgrade from 0.7.0 to newer. Can be deleted afterwards.
+        helm.sh/hook-weight: "0"
+        helm.sh/hook: "pre-install, pre-upgrade"
+        helm.sh/hook-delete-policy: "before-hook-creation"
+        helm.sh/resource-policy: "keep"
+
+  keto:
+    # https://www.ory.sh/keto/docs/reference/configuration
+    config:
+      dsn: mysql://user:password@tcp(keto-db:3306)/keto?max_conns=20&max_idle_conns=4
+      serve:
+        read:
+          port: 4466
+        write:
+          port: 4467
+      namespaces:
+        - id: 0
+          name: role
+        - id: 1
+          name: permission
+        - id: 2
+          name: participant
+
+    autoMigrate: false
```

Comment:
You can use autoMigrate instead of running migrate command manually. But its up to you.

Reply:
It wasn't working for me sadly. I don't remember the error...